crates/sui-bridge/src/types.rs

```diff
@@ -172,7 +172,7 @@ pub const ETH_TX_HASH_LENGTH: usize = 32;
 
 pub const BRIDGE_MESSAGE_PREFIX: &[u8] = b"SUI_BRIDGE_MESSAGE";
 
-#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone, Copy, TryFromPrimitive)]
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone, Copy, TryFromPrimitive, Hash)]
```

Comment:
Why do we need to add Hash to these structs?

Reply:
https://github.com/MystenLabs/sui/blob/main/crates/sui-bridge/src/server/handler.rs#L95